azurerm/resource_arm_databricks_workspace.go

```diff
@@ -0,0 +1,249 @@
+package azurerm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/databricks/mgmt/2018-04-01/databricks"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDatabricksWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDatabricksWorkspaceCreate,
+		Read:   resourceArmDatabricksWorkspaceRead,
+		Update: resourceArmDatabricksWorkspaceUpdate,
+		Delete: resourceArmDatabricksWorkspaceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"workspace_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": {
```

Comment:
There is a [`locationSchema()`](https://github.com/terraform-providers/terraform-provider-azurerm/blob/a17bf484730da0a1586893133caeb8a2d72e0476/azurerm/location.go#L9) function to declare the location field schema, you can reference the [example here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_application_insights.go#L33).